yotta/data/example-benchmark/README.md

```diff
@@ -1,100 +0,0 @@
-# mbed TLS Benchmark Example
```

Comment:
Consider moving the relevant parts here into [the new README.md](https://github.com/ARMmbed/mbed-os-example-tls/blob/master/benchmark/README.md).

Reply:
Same comment as in the previous README.md. The setup information, etc is already in the new repo, but its in the main readme instead of the example's readme. However, the last three paragraphs regarding benchmarking results are not there, so I agree that it would be good to move those three to the new repo

Reply:
I think the examples are already sufficiently well documented, as @andresag01 states, this information is in the main example `README.md`.